src/kakarot/eth_rpc.cairo

```diff
@@ -221,3 +224,86 @@ func eth_send_transaction{
 
     return result;
 }
+
+@external
```

Comment:
missing docstring. Especially that this is the unsigned data (unlike the true eth_send_raw_transaction https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_sendrawtransaction

wondering while I'm reviewing if we should then call this eth_send_raw_unsigned ?